Events/FSSAllBodiesFound.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace EddiEvents
+{
+    public class FSSAllBodiesFound : Event
```

Comment:
See above. Recommend changing the class name to `SystemScanComplete`.

Reply:
Requested change completed.